Moves the pinpad to the top of the lock screen, making it easier to avoid accidentally hitting the "Emergency Call" button when unlocking your phone.
Index: /usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
===================================================================
--- .orig/usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
+++ /usr/palm/applications/com.palm.app.phone/stylesheets/pinpad.css
@@ -14,6 +14,8 @@
 		text-align: center;
 		width: 320px;
 		color: #FFFFFF;
+		position: absolute;
+		bottom: 60px;
 }
 
 .info .big {
@@ -146,8 +148,8 @@ input.floating.single-digit {
 }
 
 .pin-main-scene .keypad-flex {
-		top: 112px;
-		bottom: 55px;
+		top: 0px;
+		bottom: 148px;
 }
 
 .pin-main-scene .keypad-flex .keypad-row {
